examples/langgraph-sls-fastapi-rag/tests/integration/test_index_graph.py

```
@@ -0,0 +1,58 @@
+import logging
+
+import dotenv
+import pytest
+from langchain_core.runnables import RunnableConfig
+
+from app.index_graph import create_graph
+from app.rag_graph import create_graph
```

Ruff (0.8.2)

Redefinition of unused `create_graph` from line 7

(F811)